package.json

```diff
@@ -3,13 +3,13 @@
   "version": "4.9.8",
   "description": "Dark theme for every website",
   "scripts": {
-    "api": "node -e \"require('./tasks/bundle-api')()\"",
+    "api": "node tasks/build.js --api",
     "benchmark-server": "node tests/benchmark-server/index.js",
-    "build": "node tasks/release.js",
+    "debug": "node tasks/build.js --build",
```

Comment:
- Sort the tasks alphabetically.
- `build.js -- build` is strange. Use `build.js --debug`.